mixer/pkg/config/crd/store.go

```diff
@@ -136,9 +135,7 @@ func (s *Store) checkAndCreateCaches(
 					ListFunc: func(options metav1.ListOptions) (runtime.Object, error) {
 						return cl.List(options)
 					},
-					WatchFunc: func(options metav1.ListOptions) (watch.Interface, error) {
-						return cl.Watch(options)
-					},
+					WatchFunc: cl.Watch,
```

Comment:
Does this capture `cl` loop variable? Why not change List at the same time?

Reply:
I think the right thing should happen with cl re capture.

cl.List is not type compatible so you need the lambda there.